### ISSUE TITLE
Clean up use of errno in twisted.internet.test.test_tcp

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8734]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8734|
|Type|enhancement|
|Created|2016-08-08 01:57:43Z|
|Branch|https://github.com/twisted/twisted/tree/8734-rodrigc-tcp-py3|

The use of errno in twisted.internet.test.test_tcp should be cleaned up

<details><summary>Searchable metadata</summary>

```
trac-id__8734 8734
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8734_rodrigc_tcp_py3 8734-rodrigc-tcp-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470621463929153 1470621463929153
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/478

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_2"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset cfd85e530c09e35be6e211176b80772eb8e7ef85
```
#!CommitTicketReference repository="" revision="cfd85e530c09e35be6e211176b80772eb8e7ef85"
Merge pull request #478 from twisted/8734-rodrigc-tcp-py3

Author: hawkowl, rodrigc
Reviewer: alex
Fixes: #8734

Fix use of errno in twisted.internet.test.test_tcp on Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
